docs/source/pages/api_domain.rst

```diff
@@ -156,17 +156,26 @@ The algorithm will choose the closest combination such as
   n\_proc=n\_col=\sqrt{nproc}
 
 In case the root is not exact the closest combitation to have :math:`n\_proc \approx n\_col` with 
-`n\_proc <  n\_col` is used. 
+`n\_proc <  n\_col` is used.
```

Comment:
I think this (also in the original) is wrong - it should be p_row not n_proc, otherwise it doesn't make sense?

Reply:
Probably yes. This is not part of the PR but I will include the fix in the PR